Rules/Playbooks/Smugglers.md

```diff
@@ -0,0 +1,105 @@
+> Copy this playbook to create a playable crew, then link back to the original. Follow the directions in [[01 Crew creation]] to get started.
+
+## Crew Stats
+
+### reputation
+
+### lair
+
+### trackers
+
+| capital       | rep          | turf        | tier        | hold        | XP          |
+| ------------- | ------------ | ----------- | ----------- | ----------- | ----------- |
+| `clock 2 / 4` | `clock 0/12` | `clock 0/6` | `clock 0/5` | `clock 1/1` | `clock 0/9` |
+_turf counts as "permanent" rep. reduce the length of the rep track by 1 for each turf you have marked._
+
+See the [[15 The Faction Game#Advancement|faction advancement]] rules for how to increase your tier.
+
+**You gain XP when...**
+- You meet one of the [[16 Advancement#Crew Advancement|generic advancement criteria]]
+- You **\<add a crew specific advancement trigger>**
+## Exposure
+
+| heat        | wanted level |
+| ----------- | ------------ |
+| `clock 0/9` | `clock`      |
+See the [[03 Heat|heat]] rules for how you manage heat and the consequences it can have.
+## Preferred Cargo
+
+> Choose a preferred cargo from this list and delete the rest.
+- weapons
+- people
+- contraband
+- alien technology
```

Comment:
I forget to what degree alien technology is a catch all, but I could see biological material, hazardous or otherwise being on this list.

Reply:
I had been thinking of alien tech as a catchall. Bio material seems like it potentially fit into any of these tbh.